packaging/command-line-assistant.spec

```diff
@@ -58,8 +56,18 @@ RHEL systems.
 Summary:    CLAD SELinux policy
 BuildArch:  noarch
 
+# Set selinux_ver depending on RHEL version
+%if 0%{?rhel} && 0%{?rhel} < 10
+# RHEL 9.7 and 10.1 GA versions
```

Comment:
This comment isn't exactly accurate, right? The condition above is going to capture on RHEL 9, but not RHEL 10.

Reply:
```suggestion
```

yeah, think I can remove it